### PR TITLE
add v1.9

### PR DESCRIPTION
**Pull request by mrIncompetent**
**What this PR does / why we need it**:
Adds kubernetes 1.9 support.

### CODE REVIEW

File: addon-manager/addons/dashboard/dashboard-controller.yaml [hunk 1]
```diff
@@ -1,49 +1,66 @@
...
+        resources:
+          limits:
+            cpu: 100m
+            memory: 300Mi
+          requests:
+            cpu: 50m
+            memory: 100Mi
         ports:
-        - containerPort: 9090
+        - containerPort: 8443
```
Comment (metalmatze) [inline addon-manager/addons/dashboard/dashboard-controller.yaml:41]: Why did you change this from 9090 to 8443?

Comment (mrIncompetent) [reply addon-manager/addons/dashboard/dashboard-controller.yaml:41]: upstream change - i would like to keep as close as possible to upstream

Comment (metalmatze) [reply addon-manager/addons/dashboard/dashboard-controller.yaml:41]: Sure. Just wanted to know. Probably because they changed the internal one from 80 to 443. :+1:

Comment (mrIncompetent) [reply addon-manager/addons/dashboard/dashboard-controller.yaml:41]: Yes, they switched to https. A certificate will be created during startup

**Review by metalmatze — APPROVED** [review]
